docker/table_docker_container.go

```diff
@@ -35,6 +35,8 @@ func tableDockerContainer(ctx context.Context) *plugin.Table {
 			{Name: "host_config", Type: proto.ColumnType_JSON, Description: "Host configuration for the container."},
 			{Name: "network_settings", Type: proto.ColumnType_JSON, Description: "Network settings for the container."},
 			{Name: "mounts", Type: proto.ColumnType_JSON, Description: "Volume mounts for the container."},
+			{Name: "config", Type: proto.ColumnType_JSON, Description: "Config contains the configuration data about a container.", Hydrate: getContainerInspect},
```

Comment:
@bigdatasourav If `config` is a top level property from the inspect data, can you please share an example of what the full `inspect` data looks like? I'm curious as to why only `config` was pulled into a separate column, but not others

Reply:
@cbruno10, below is the structure of inspect - 
```
type ContainerJSON struct {
	*ContainerJSONBase
	Mounts          []MountPoint
	Config          *container.Config
	NetworkSettings *NetworkSettings
}
```
We already have the Mounts and NetworkSettings columns from the list API.

Reply:
@bigdatasourav Ah ok, that makes sense then, especially since we already have the `mounts` and `network_settings` columns. I'm good with adding the `config` column then.